src/natcap/invest/validation.py

```diff
@@ -844,9 +846,6 @@ def get_headers_to_validate(spec):
     """
     headers = []
     for key, val in spec.items():
-        # for now only check headers that are always required
-        # assume that any conditionally-required headers are validated by the
-        # model's validate function
         # if 'required' isn't a key, it defaults to True
         if ('required' not in val) or (val['required'] is True):
```

Comment:
At this point, will `required` always be a bool? If so, could simplify to
```suggestion
        if val['required']:
```

Reply:
Unfortunately no, we haven't fully cleaned the SPEC by this point in `validate`, so `required` could still be absent.  We have so far only handled the case where `required` is present and is a `str`, which we have rewritten as a bool.